internal/topology/variables/clusterclass_variable_validation_test.go

```diff
@@ -631,6 +632,76 @@ func Test_ValidateClusterClassVariable(t *testing.T) {
 				},
 			},
 		},
+		{
+			name: "fail if field is required below properties and sets a default",
```

Comment:
```suggestion
			name: "fail if field is required below properties and sets a default that misses the field",
```
nit: up to you. Feel free to resolve if you prefer the current shorter version